governance/veANGLE/interests.md

```diff
@@ -0,0 +1,28 @@
+---
+description: Details about how veANGLE can boost ANGLE rewards for LPs
+---
+
+# 📈 Interests for veANGLE holders
+
+One benefit of owning veANGLE is the redistribution of part of the interests earned by the protocol.
+
+Until the upgrade, the interests generated were redistributed to SLPs as profit, and to the protocol as surplus/reserve. Now, they are also shared to veANGLE holders as an incentive for taking part in the governance of the protocol.
+
+![Angle Interests Redistribution](../../.gitbook/assets/angle-interests-redistribution.png)
+
+## Interests redistribution
+
+### Share of interests for veANGLE holders
+
+The share of interests redistributed to veANGLE holders is a parameter that can be set and updated by Governance. The goal is that this parameter is high enough to bring added value to veANGLE holders, but not too high to damage the reserves being built by the protocol and the yield earned by SLPs.
+
+### Redistribution
+
+The protocol will distribute interests in the form of Angle tokens, beginning by redistributing sanUSDC_EUR.
```

Comment:
We should also explain here the redistribution flow: like how it's going to work for FEI, FRAX and DAI -> converted to USDC and then deposited